RedHat-UX/red-hat-design-system
# fix(alert): styling header font-weight and vertical alignment

Closes #920 

## What I did

1.  Added in slotted styles to make the alert header `bold` (700 font weight)
2. Added styles to help with the alignment of the icon with the header text


## Testing Instructions

1.  Please check the Deploy Preview and confirm that the alerts look good in the docs and the demos
